src/simplification/Simplifier.ts

```diff
@@ -86,6 +50,52 @@ export class Simplifier {
     return [model, ...modelsToReturn];
   }
 
+  /**
+   * Function to simplify all model properties from schema.
+   * 
+   * @param model to simplify properties to 
+   * @param schema to simplify
+   */
+  private simplifyModelProperties(model: CommonModel, schema: Schema) {
```

Comment:
I think that `simplifyModel` is enough. By `simplifyModelProperties` I have in mind `object` type and this is not a true - we also simplify here items, enums etc.

Reply:
Agreed 😄